packages/Pager/src/Pager.tsx

```diff
@@ -153,7 +159,7 @@ const Pager: React.FunctionComponent<PagerProps> = ({
             </nav>
             <div className="ids-pager__results">
                 {pageSize * currentPage - pageSize + 1}-
-                {Math.min(pageSize * currentPage, totalCount)} of {totalCount}
+                {Math.min(pageSize * currentPage, totalCount)} {stringFormatter.format("of")} {totalCount}
```

Comment:
Might not be that bad in this scenario, but i think we try to offer translation for aria-labels AND offer defaults that make sense for string that are visually rendered. But the consumers should always be able to override the strings. 

for instance, in this case, a pager should offer a method that takes 3 parameters and return a string or ReactNode, and we offer the default value which would be the string template

Reply:
React-spectrum for instance offer a "valueLabel" props

https://react-spectrum.adobe.com/react-aria/Meter.html#props

valueLabel - ReactNode- The content to display as the value's label (e.g. 1 of 4).
showValueLabel - boolean- Whether the value's label is displayed. True by default if there's a label, false by default if not.

Reply:
so from what I understand, you want me to add a prop like renderResultsText that passes the pageSize, currentPage and totalCount?
If not set, it defaults to our default message?

Reply:
They have all the informations already, so the function is overkill.

add to the interface
```
valueLabel?: ReactNode
```

and we set it's default value in our codebase if it's null
```
valueLabel = stringFormatter.format("pageRange", { minRange: pageSize * currentPage - pageSize + 1, maxRange: Math.min(pageSize * currentPage, totalCount), totalCount: totalCount })
```

you can chose variable name that makes more sense in the context (minRange,maxRange,totalCount)